addons/crm/wizard/crm_lead_to_opportunity_mass.py

```diff
@@ -29,7 +28,7 @@ def default_get(self, fields):
     action = fields.Selection(selection_add=[
         ('each_exist_or_create', 'Use existing partner or create'),
     ], string='Related Customer', required=True)
-    force_assignation = fields.Boolean('Force assignation', help='If unchecked, this will leave the salesman of duplicated opportunities')
+    force_assignemnt = fields.Boolean(default=False)
```

Comment:
why remove the help ?
Also, don't we prefer to always add a string for every field ?

Reply:
Actually it comes from inherit. It has been added on crm_lead2opportunity model that is inherited from this one.